semgrep-core/Synthesizing/Pattern_from_Targets.mli

```diff
@@ -0,0 +1,63 @@
+
+(*
+ * The pattern_from_targets feature allows users to highlight separate code
```

Comment:
Great comment! I think it's better to keep the long detailed explanation in the .ml and just comments on the interface and general purpose in the .mli.